tests/spread/ci/build-image/task.yaml

```diff
@@ -9,17 +9,15 @@ execute: |
     cd $PROJECT_PATH
 
     # install prerequisites
-    apt update -qq
-    apt install retry -yqq
     snap install google-cloud-sdk --classic
     snap install lxd --channel=latest/stable
     lxd init --auto
 
     # authenticate with gcloud
     gcloud auth activate-service-account --key-file="./sa.json"
 
-    # launch the new lxc instance from an ubuntu 20.04 base
-    lxc launch ubuntu:20.04 $INSTANCE_NAME
+    # launch the new lxc instance from an ubuntu 22.04 base
+    lxc launch images:ubuntu/jammy $INSTANCE_NAME
```

Comment:
I know we have to use the `ubuntu/jammy` image because `22.04` is not released yet, but I was wondering, what will be the difference when that happens? If not exactly the same we need to decide which one we would use after the release.

Reply:
I unfortunately don't know this, I'm using ubuntu/jammy for the reason you said, but I don't know what will happen or if we will need to update this. Maybe someone else can answer this?

Reply:
I added the TODO